en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -97,97 +99,180 @@ In this tutorial, let's design the REST API by updating the low-code diagram as
-        | Expression*       | `<decimal>(totalCases / populationMillions)` |
-
-    3. Click **Save**.
-
-9. To build the JSON payload with data of the total cases per million in the population, add a variable.
-
-    Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
-
-    | **Field**         | **Value**                     |
-    |-------------------|-------------------------------|
-    | **Variable Type** | `json`                        |
-    | **Variable Name** | `payload`                     |
-    | Expression        | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
-
-    Click **Save**.
-
-10. To respond with the JSON payload, add a `Return` statement as follows:
-
-    1. Click the last **+** icon in the low-code diagram and click **Return**.
-
-    2. In the **Return Expression** field, enter `payload`.
-
-    3. Click **Save**.
+   1. Click **+** under **Start**.
+   
+   2. In the **Add Constructs** pane that appears, click **Connector**.
+
+   3. Search for **COVID-19**, and select the **COVID-19 by ballerinax** connector when it appears in the search results. 
+   
+       In the **Endpoint** pane that appears, click **Save** to add the endpoint.
+   
+   4. To add a query parameter to the connector you added, click **+** below it.
+   
+       In the **Add Constructs** pane, click **Action**. The **Action** pane opens. Configure the action as follows:
+
+       1. In the **Action** pane, click **covid19Ep**.
+      
+       2. Click **getStatusByCountry**.
+      
+       3. In the action statement, double-click **`getStatusByCountryResponse`** variable name to edit it. Replace the default value with `statusByCountry`.
+      
+       4. In the **Parameters** tab, check whether the **country** check box is selected. If not, select it.
+      
+          This adds `country` as a query parameter in the connector configuration to allow the REST API to fetch COVID-19 statistics specific to a given country.
+        
+       5. Click **Save**.
+        
+   5. To extract the response of the **COVID-19** connector, add a variable by following these sub-steps.
+   
+       1. Click the last **+** in the low-code diagram.
+      
+       2. Click **Variable**.
+      
+          In the **Variable** pane, configure the variable as follows:
+      
+           1. On the variable statement, click **`var`**, and click **int** on the list that appears below in the **Suggestions** tab. Here, you are changing the variable type from `var` to `int`.
+         
+           2. To edit the variable name, double-click **`variable`**. Then enter `totalCases` as the variable name.
+         
+           3. Click  **`<add-expression>`** and click **statusByCountry** in the list that appears below in the **Suggestions** tab. Here, you are selecting the **statusByCountry** query parameter you previously added to derive information related to a specific country.
```

Comment:
Should we go with a more simplified sentence similar to the following?
Click  **`<add-expression>`** and then click **statusByCountry** displayed in the list of suggestions.

Reply:
Changed as suggested.